app/repository/FeatureRepository.scala

```diff
@@ -82,9 +94,10 @@ class FeatureRepository @Inject()(db: Database, tagRepository: TagRepository, sc
   }
 
   def findAllByBranchId(branchId: Long): Seq[Feature] = {
-    db.withConnection { implicit connection =>
+    val s =  db.withConnection { implicit connection =>
```

Comment:
Revert this change as this local variable can be inlined
```suggestion
      db.withConnection { implicit connection =>
```